hazelcast-spring/src/test/resources/com/hazelcast/spring/node-client-applicationContext-hazelcast.xml

```diff
@@ -43,7 +43,7 @@
                 <hz:property name="hazelcast.merge.first.run.delay.seconds">5</hz:property>
                 <hz:property name="hazelcast.merge.next.run.delay.seconds">5</hz:property>
             </hz:properties>
-            <hz:network port="${cluster.port}" port-auto-increment="false">
+            <hz:network port="${cluster.port}" port-auto-increment="true">
```

Comment:
This is the only change I cannot fully agree or disagree with, since I don't know if it was intentional to test it without auto-incrementing ports.

Reply:
It's here by mistake. Without it i can't run the `TestClientApplicationContext` locally on Windows because port 5700 is used by the system, and the test was failing all the time on PR builder. However, the test is passing just fine with this change, so I'd like to keep it here. I don't think this test relies on the fact that the member is on port 5700, I guess it's a copy-paste from another test - there are tests which assert this.